packages/number-field/src/number-field.css

```diff
@@ -13,6 +13,41 @@ governing permissions and limitations under the License.
 @import url('./spectrum-number-field.css');
 @import url('./number-field-overrides.css');
 
+:host {
+    --spectrum-stepper-width: calc(
+        var(--mod-stepper-height, var(--spectrum-stepper-height)) *
+            var(
+                --mod-stepper-min-width-multiplier,
+                var(--spectrum-text-field-minimum-width-multiplier)
+            ) +
+            var(
+                --mod-stepper-button-width,
+                var(--spectrum-stepper-button-width)
+            ) +
+            var(
+                --mod-stepper-border-width,
+                var(--spectrum-stepper-border-width)
+            ) * 2
+    );
+
+    inline-size: var(--mod-stepper-width, var(--spectrum-stepper-width));
+}
```

Comment:
When inspecting `sp-number-field` in the browser, this calc is undefined. It seems like `--spectrum-stepper-height` and `--spectrum-stepper-button-width` are undefined, which then renders `--spectrum-stepper-width` and the `inline-size` styles undefined. 

What should I be expecting there? Or is that expected, for them not to be defined on `host`? I noticed this when I was testing `--mod-stepper-width`, which works just fine. 👍 

https://github.com/user-attachments/assets/b074e8f7-add3-46cb-bd43-787f715e78c0